strapi/strapi
# fix: Description and min/max characters not visible during error screen

Fixes https://github.com/strapi/strapi/issues/16290

### What does it do?
- Unclear max/min error messages with incorrect input length

### Why is it needed?
- Visibility of the minimum / maximum characters and description during the error.

### How to test it?
1. Add a collection type with a text field and set minimum / maximum characters and add a description.
2. Add content within this new collection type
3. Notice the minimum/maximum characters and description below the field.
4. Add less or more characters in the field.
5. Publish the content
6. The error message is descriptive enough to correct that error.


## Demo
![Screenshot 2023-04-14 at 6 40 27 PM](https://user-images.githubusercontent.com/45232708/232118537-1ff7d475-be47-4752-b747-ad4760ca5734.png)
